modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,271 @@
+                atol=0.011,
+            )
+
+    predictions = bst.predict(dmatrix_xgb)
+    modin_predictions = modin_bst.predict(dmatrix_mxgb, num_actors=num_actors)
+
+    preds = pd.DataFrame(predictions).apply(lambda x: round(x))
+    modin_preds = modin_predictions.apply(lambda x: round(x))
+
+    val = accuracy_score(y, preds)
+    modin_val = accuracy_score(modin_y, modin_preds)
+
+    np.testing.assert_allclose(val, modin_val, atol=0.002, rtol=0.002)
+
+
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_iris(),
+            {
+                "num_class": 3,
+            },
+        ),
+        (
+            load_digits(),
+            {
+                "num_class": 10,
+            },
+        ),
+        (
+            load_wine(),
+            {
+                "num_class": 3,
+            },
+        ),
+    ],
+    ids=[
+        "load_iris",
+        "load_digits",
+        "load_wine",
+    ],
+)
+def test_dmatrix_train_predict_multiclass_classification_datasets(data, num_actors):
+    dataset, param_ = data
+    num_round = 10
+    part_param = {"objective": "multi:softprob", "eval_metric": "mlogloss"}
+    param = {**param_, **part_param}
+
+    X = dataset.data
+    y = dataset.target
+    dmatrix_xgb = xgboost.DMatrix(X, label=y)
+
+    modin_X = pd.DataFrame(X)
+    modin_y = pd.Series(y)
+    dmatrix_mxgb = xgb.DMatrix(modin_X, label=modin_y)
+
+    evals_result_xgb = {}
+    evals_result_mxgb = {}
+    verbose_eval = False
+    bst = xgboost.train(
+        param,
+        dmatrix_xgb,
+        num_round,
+        evals_result=evals_result_xgb,
+        evals=[(dmatrix_xgb, "train")],
+        verbose_eval=verbose_eval,
+    )
+    modin_bst = xgb.train(
+        param,
+        dmatrix_mxgb,
+        num_round,
+        evals_result=evals_result_mxgb,
+        evals=[(dmatrix_mxgb, "train")],
+        num_actors=num_actors,
+        verbose_eval=verbose_eval,
+    )
+
+    assert len(evals_result_xgb["train"]["mlogloss"]) == len(
+        evals_result_mxgb["train"]["mlogloss"]
+    )
+    for i in range(len(evals_result_xgb["train"]["mlogloss"])):
+        np.testing.assert_allclose(
+            evals_result_xgb["train"]["mlogloss"][i],
+            evals_result_mxgb["train"]["mlogloss"][i],
+            atol=0.009,
+        )
+
+    predictions = bst.predict(dmatrix_xgb)
+    modin_predictions = modin_bst.predict(dmatrix_mxgb, num_actors=num_actors)
+
+    array_preds = np.asarray([np.argmax(line) for line in predictions])
+    modin_array_preds = np.asarray(
+        [np.argmax(line) for line in modin_predictions.to_numpy()]
+    )
+
+    val = accuracy_score(y, array_preds)
+    modin_val = accuracy_score(modin_y, modin_array_preds)
+
+    np.testing.assert_allclose(val, modin_val)
+
+
+@pytest.mark.parametrize(
+    "num_actors",
+    [
+        1,
+        pytest.param(num_cpus, marks=pytest.mark.skip),
```

Comment:
Do we need to add one more value modin.config.NPartitions.get() + 1?
Why are the two cases skipped?

Reply:
@YarShev https://github.com/modin-project/modin/issues/3242

Reply:
Why don't the others tests fail?

Reply:
In other tests uses only train_data in train(). In this test uses train_data and test_data in train()

Reply:
Discussed offline. Decided to leave as it is.